tests/func/test_data_cloud.py

```diff
@@ -507,18 +507,18 @@ def _test_cloud(self, remote=None):
         cache_dir = stage_dir.outs[0].cache_path
 
         # FIXME check status output
-        self.main(["status", "-c", "--show-checksums"] + args)
+        #        self.main(["status", "-c", "--show-checksums"] + args)
```

Comment:
If this isn't needed it is better to delete it than commenting it (the same for the comments below)